scripts/build.bat

```diff
@@ -0,0 +1,53 @@
+@echo off
+rem Please do not modify this script. Copyt into the build folder
+
+rem Set compiler
+set CC=cl.exe
+set CXX=cl.exe
+
+rem Set variables
+if not defined CMAKE_GENERATOR set CMAKE_GENERATOR=Visual Studio 16 2019
+if not defined NUMBER_OF_COMPILATION_CORES set NUMBER_OF_COMPILATION_CORES=%NUMBER_OF_PROCESSORS%
+if not defined COSIMIO_SOURCE set COSIMIO_SOURCE=%~dp0..
+if not defined COSIMIO_BUILD set COSIMIO_BUILD=%COSIMIO_SOURCE%/build
+
+rem Set defaults
+if not defined CO_SIM_IO_BUILD_TESTING set CO_SIM_IO_BUILD_TESTING=ON
+if not defined CO_SIM_IO_BUILD_C set CO_SIM_IO_BUILD_C=OFF
+if not defined CO_SIM_IO_BUILD_PYTHON set CO_SIM_IO_BUILD_PYTHON=OFF
+if not defined CO_SIM_IO_BUILD_FORTRAN set CO_SIM_IO_BUILD_FORTRAN=OFF
+if not defined CO_SIM_IO_STRICT_COMPILER set CO_SIM_IO_STRICT_COMPILER=ON
```

Comment:
this seems superfluous, it is already set in cmake

Reply:
True, I added for debugging poruposes